lib/index.js

```diff
@@ -54,20 +65,23 @@ class PWMetrics {
           results[index] = data;
         }).catch(error => {
           console.error(getMessageWithPrefix('ERROR', 'FAILED_RUN', index, results.length, error.message));
-          throw new Error(error);
+          console.error(error.stack + '\n');
```

Comment:
But using `console.error(error.stack + '\n');` you output the trace and it feels like error throw, which is not. If you wanna show error and continue process, then what about consoling just error message without trace to avoid confusing.